frontend-react/src/utils/TemporarySettingsAPITypes.ts

```diff
@@ -0,0 +1,278 @@
+    INACTIVE = "inactive",
+    TESTING = "testing",
+    ACTIVE = "active",
+}
+
+enum ProcessingType {
+    SYNC = "sync",
+    ASYNC = "async",
+}
+
+enum ReportStreamFilterDefinition {
+    BY_COUNTY = "filterByCounty",
+    MATCHES = "matches",
+    NO_MATCH = "doesNotMatch",
+    EQUALS = "orEquals",
+    VALID_DATA = "hasValidDataFor",
+    AT_LEAST_ONE = "hasAtLeastOneOf",
+    ALLOW_ALL = "allowAll",
+    ALLOW_NONE = "allowNone",
+    VALID_CLIA = "isValidCLIA",
+    DATE_INTERVAL = "inDateInterval",
+}
+
+enum BatchOperation {
+    NONE = "NONE",
+    MERGE = "MERGE",
+}
+
+enum EmptyOperation {
+    NONE = "NONE",
+    SEND = "SEND",
+}
+
+enum USTimeZone {
+    PACIFIC = "US/Pacific",
+    MOUNTAIN = "US/Mountain",
+    ARIZONA = "US/Arizona",
+    CENTRAL = "US/Central",
+    EASTERN = "US/Eastern",
+    SAMOA = "US/Samoa",
+    HAWAII = "US/Hawaii",
+    EAST_INDIANA = "US/East-Indiana",
+    INDIANA_STARKE = "US/Indiana-Starke",
+    MICHIGAN = "US/Michigan",
+    CHAMORRO = "Pacific/Guam",
+}
+
+enum FtpsProtocol {
+    SSL = "SSL",
+    TLS = "TLS",
+}
+
+enum GAENUUIDFormat {
+    PHONE_DATE = "PHONE_DATE",
+    REPORT_ID = "REPORT_ID",
+    WA_NOTIFY = "WA_NOTIFY",
+}
+
+type ReportStreamSettingsEnum =
+    | "jurisdiction"
+    | "format"
+    | "customerStatus"
+    | "processingType"
+    | "reportStreamFilterDefinition";
+
+const getListOfEnumValues = (e: ReportStreamSettingsEnum): string[] => {
+    switch (e) {
+        case "customerStatus":
+            return Array.from(Object.values(CustomerStatus));
+        case "format":
+            return Array.from(Object.values(Format));
+        case "jurisdiction":
+            return Array.from(Object.values(Jurisdiction));
+        case "processingType":
+            return Array.from(Object.values(ProcessingType));
+        case "reportStreamFilterDefinition":
+            return Array.from(Object.values(ReportStreamFilterDefinition));
+    }
+};
+
+abstract class SampleObject {
+    stringify() {
+        return JSON.stringify(this, null, 6);
+    }
+    abstract getAllEnums(): Map<string, string[]>;
+    abstract description(): string;
+}
+
+class SampleFilterObject extends SampleObject {
+    filters = [
+        {
+            topic: "covid-19",
+            jurisdictionalFilter: [],
+            qualityFilter: [],
+            routingFilter: [],
+            processingModeFilter: [],
+        },
+    ];
+
+    stringify(): string {
+        return JSON.stringify(this.filters, null, 6);
+    }
+
+    getAllEnums(): Map<string, string[]> {
+        return new Map<string, string[]>([
+            [
+                "Available Filters",
+                Array.from(Object.values(ReportStreamFilterDefinition)),
+            ],
+        ]);
+    }
+
+    description(): string {
+        return "This field takes an array of filter objects (see object above). Click this tooltip to copy the sample value.";
+    }
+}
+
+class SampleJwkSet {
+    scope = "scope";
+    keys = {
```

Comment:
This just seems weird. These are internal keys for JWK... why are they listed? They are usually opaque.

I'd like to understand and maybe we can add a comment explaining this section.

Reply:
So, the idea behind this ticket is to provide sample json objects that can be copy/pasted into the field and work. And if they want to add a JwkSet to the settings, they'll need to have valid Jwk fields. These objects are all Sample objects; they're not meant to be used to store data in the application, they're just an easier way of generating a bunch of sample json shapes in a tooltip.

Reply:
OK. I guess that makes sense, it's just feels weird.